### PR TITLE
feat(website): getting started FAQs

### PR DESCRIPTION
**Pull request by SiTaggart**
Add an FAQ page to getting started.

Fix a few sidebar navigation items.

Fix a link to font css file

### CODE REVIEW

**Review by richbachman — APPROVED** [review]
Code and content for the engineering FAQ look good to me.

File: packages/paste-website/src/pages/getting-started/faqs/index.mdx [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+## Engineering
+
+<Stack orientation="vertical" spacing="space140">
+  <Disclosure visible>
+    <DisclosureHeading as="h3" variant="heading30">
+      What browser support does Paste have?
+    </DisclosureHeading>
+    <DisclosureContent>
+      <Paragraph>
+        We support the latest and latest minus one versions of all modern browsers on the market today. Most notible
```
Comment (SiTaggart) [inline packages/paste-website/src/pages/getting-started/faqs/index.mdx:56]:
```suggestion
        We support the latest and latest minus one versions of all modern browsers on the market today. Most notable
```

File: packages/paste-website/src/pages/getting-started/faqs/index.mdx [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+        Whilst we are acutely aware that a Design System cannot do and hold everything that is needed for each
+        individual product, and that extension and customization is the key to a successful Design System at scale, we
+        will not be providing that functionality via a custom classname.
+      </Paragraph>
+      <Paragraph>We won't be doing this for a number of reasons:</Paragraph>
+      <UnorderedList>
+        <ListItem>
+          Global classnames, especially overrides, impact the confidence teams can have on the state of their styling.
+          There is no guarantee you won't affect someone else or they won't affect you, impacting the success of
+          independent teams and confidence in your deploys. This particularly applies to a number of Twilio products
```
Comment (SiTaggart) [inline packages/paste-website/src/pages/getting-started/faqs/index.mdx:83]:
```suggestion
          independent teams and confidence in your deployments. This particularly applies to a number of Twilio products
```

File: packages/paste-website/src/pages/getting-started/faqs/index.mdx [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+      </Paragraph>
+    </DisclosureContent>
+  </Disclosure>
+  <Disclosure>
+    <DisclosureHeading as="h3" variant="heading30">
+      Why does Paste use React-uid to generate ids? Why don't you use static ids?
+    </DisclosureHeading>
+    <DisclosureContent>
+      <Paragraph>
+        ID references in a webpage must be unique for them to work correctly, especially when supporting assistive
```
Comment (SiTaggart) [inline packages/paste-website/src/pages/getting-started/faqs/index.mdx:108]:
```suggestion
        ID references in a web page must be unique for them to work correctly, especially when supporting assistive
```

File: packages/paste-website/src/pages/getting-started/faqs/index.mdx [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+        technologies such as screen readers. In a multi tenant, single page application you do not always have control
+        of what is loaded into a webpage at any one time. As a result static ids can clash and cause duplicates.
+      </Paragraph>
+      <Paragraph>
+        To guarantee that every id in a webpage is unique, regardless of what was loaded on the page and in what order,
```
Comment (SiTaggart) [inline packages/paste-website/src/pages/getting-started/faqs/index.mdx:113]:
```suggestion
        To guarantee that every id in a web page is unique, regardless of what was loaded on the page and in what order,
```

File: packages/paste-website/src/pages/getting-started/faqs/index.mdx [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+        we use react-uid to generate ids for us.
+      </Paragraph>
+    </DisclosureContent>
+  </Disclosure>
+  <Disclosure>
+    <DisclosureHeading as="h3" variant="heading30">
+      OK, so you don't support classNames or static IDs, how do I create DOM references for my tests?
+    </DisclosureHeading>
+    <DisclosureContent>
+      <Paragraph>We recommend using data attributes for your test indentifiers.</Paragraph>
```
Comment (SiTaggart) [inline packages/paste-website/src/pages/getting-started/faqs/index.mdx:123]:
```suggestion
      <Paragraph>We recommend using data attributes for your test identifiers.</Paragraph>
```

File: packages/paste-website/src/pages/getting-started/faqs/index.mdx [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+    <DisclosureContent>
+      <Paragraph>
+        Paste leaves loading the font files up to the consumer of the system, to accomodate any technical or licensing
+        restrictions a product might have.
+      </Paragraph>
+      <Paragraph>
+        For the default theme Paste uses Inter. This is available as a{' '}
+        <Anchor href="https://fonts.google.com/specimen/Inter">Google web font</Anchor> can be loaded in a number of
+        ways into your application. Please choose the one that best suits your preferred choice of asset delivery, or
+        follow how{' '}
```
Comment (SiTaggart) [inline packages/paste-website/src/pages/getting-started/faqs/index.mdx:168]:
```suggestion
        follow{' '}
```

File: packages/paste-website/src/pages/getting-started/faqs/index.mdx [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+        <Anchor href="https://github.com/twilio-labs/paste/blob/main/packages/paste-website/src/assets/css/fonts.css#L105">
+          how we load the font
+        </Anchor>{' '}
+        on this website.
+      </Paragraph>
+      <Paragraph>
+        For the Console theme, due to licensing restictions only web properties under certain domains are allowed to
```
Comment (SiTaggart) [inline packages/paste-website/src/pages/getting-started/faqs/index.mdx:175]:
```suggestion
        For the Console theme, due to licensing restrictions only web properties under certain domains are allowed to
```

**Pull request merged**